### PR TITLE
Video listing

### PR DESCRIPTION
**Pull request by BelideNikhil**
## Features

+ Video Listing Page
+ Filter videos by category

### CODE REVIEW

File: src/Components/VideoCard/VideoCard.css [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    font-weight: var(--fw--semi-bold-500);
+    color: var(--clr-accent-one);
+    -webkit-line-clamp: 2;
+    -webkit-box-orient: vertical;
+    display: -webkit-box;
+    overflow: hidden;
+}
+.creator-img {
+    width: 36px;
+    height: 36px;
```
Comment (bipin7yadav) [inline src/Components/VideoCard/VideoCard.css]: could have used rem instead of px

**Review by bipin7yadav — APPROVED** [review]
code looks clean

**Review by gadgetfather — APPROVED** [review]
- Code is clean 
- Good work on UI 
- one thing that I noticed is, that I was not able to see the loader. I would suggest instead of using SVG, create a loader component and use that
here is the link for your reference:
https://projects.lukehaas.me/css-loaders/

File: src/Components/VideoCard/VideoCard.css (hunk 1 above)
Comment (BelideNikhil) [reply src/Components/VideoCard/VideoCard.css]: done!
changed it to rem

**Comment by BelideNikhil** [comment]
> * Code is clean
> * Good work on UI
> * one thing that I noticed is, that I was not able to see the loader. I would suggest instead of using SVG, create a loader component and use that
>   here is the link for your reference:
>   https://projects.lukehaas.me/css-loaders/

Thanks for the loader suggestion, replaced the SVG with CSS Loader as suggested

**Pull request merged**